src/FieldDescription/FieldDescriptionFactory.php

```diff
@@ -49,7 +49,7 @@ public function create(string $class, string $name, array $options = []): FieldD
      * @phpstan-param class-string $baseClass
      *
      * @phpstan-return array{
-     *      ClassMetadata,
+     *      ClassMetadata<object>,
```

Comment:
isnt this related to the class string above?

Reply:
Not always, with `foo.bar.baz`, you give Foo, you and with ClassMetadata of Baz.